File: Desktop/Domain/CmdMsgs/DTMFCmd.cs
using System;

using System.Xml;
using System.Xml.Xsl;
using System.Collections;
using System.Runtime.InteropServices;

namespace Blurts
{
  [ComVisible(true)]
  public class DTMFCmd : CmdBase
  {
    // Tag Names
    private static String KEY_DTMF = "DTMFTones";


    public DTMFCmd(String s)
      : base(s)
    {
    }

    public DTMFCmd()
      : base(CMD_DTMF)
    {
    }

    [ComVisible(true)]
    public String DTMFTones
    {
      get
      {
        return optString(KEY_DTMF);
      }
      set
      {
        try
        {
          remove(KEY_DTMF);
          put(KEY_DTMF, value);
        }
        catch (JSONException e)
        {
          Console.WriteLine(e.ToString());
        }
      }
    }
  }
}

File: BlackBerry/AlertMsgs/DTMFCmd.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class DTMFCmd extends CmdBase 
{
  // Tag Names
  private static final String KEY_DTMF = "DTMFTones";


  public DTMFCmd( String string ) throws JSONException
  {
    super( string );
  }

  public DTMFCmd()
  {
    super( CMD_DTMF );
  }

  String getDTMFTones()
  {
    return optString( KEY_DTMF );
  }
}
